index.html

```diff
@@ -9,25 +10,77 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Awesomebooks</title>
   <link rel="stylesheet" href="./styles/styles.css">
+  <script src="https://cdn.jsdelivr.net/npm/luxon@2.1.1/build/global/luxon.min.js"></script>
 </head>
+
 <body>
+  <main class="container">
+
+
+    <nav class="toolbar">
+      <a href="./index.html">Awesome Books</a>
 
-  <section class="container">
-  <h1 id="awesomebookstitle">Awesome Books</h1>
-  <div id="store">
-    <div class="empty hide" id="empty-id">
-      
-    </div>
-  </div>
-  
-  <h2 id="addnewbooktitle">Add a new Book</h2>
-  <form action="#" id="form-id">
-    <input type="text" name="title" id="title" placeholder="Add Book Title" required>
-    <input type="text" name="author" id="author" placeholder="Add Book Author" required>
-    <button id="add-btn" type="submit" >Add</button>
-  </form>
-</section>
+      <ul class="navlinks">
+        <li class="link">
+          <a href="#bookStore" id="library">Book List</a>
+        </li>
+        <li class="link">
+          <a href="#addbook" id="addLink">Add Book</a>
+        </li>
+        <li class="link">
+          <a href="#contact" id="contactLink">Contact</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="dateAndTime" id="dateDisplay">sdf</div>
+    <section id="bookStore">
+      <div id="store">
+        <div class="empty hide" id="empty-id">
+        </div>
```

Comment:
- [ ] Please as requested by the previous reviewer, try and follow the wireframe in displaying the list of books. The wireframe displays details of the book horizontally and your design displayed them vertically. Kindly fix that please.
### Your design
![1](https://user-images.githubusercontent.com/94551484/199827426-dcdf8a79-2327-49f9-be61-4ca79c4ed675.png)
### Wireframe
![f1](https://user-images.githubusercontent.com/94551484/199827457-0cd370bd-24bc-4f82-9637-dce6f71640e2.png)